crates/matrix-sdk/src/widget_api/messages/from_widget/actions.rs

```diff
@@ -0,0 +1,25 @@
+use serde::{Deserialize, Serialize};
+
+use url::Url;
+
+use crate::widget_api::messages::message::{Message, ActionBody};
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+#[serde(tag = "action")]
+pub enum FromWidgetAction {
```

Comment:
Should we maybe call it `WidgetMessage`? I.e. `From` could probably be omitted here given that the `from` is part of the module name, so the user has this implied (there is a non-written "rule" that if the module contains the "prefix" for the structure, it could be omitted since it's clear for the caller that it's a `FromXXX` message.

Reply:
This would result in there beeing two WidgetMessage enums with the same name. Of course they are in different modules but it still would be confusing when looking at the file and not immediatly knowing if its from or to widget. And they are already easy to get confused.
So i think in this case I would be unecassary explicit